src/posts/blog/pactfi-aeneas-rewards-degen/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: 'PactFi Adds Degen to Aeneas Rewards'
+date: '04/11/2022'
+---
+
+import Post from '../../../components/post';
+
+<Post frontmatter={props.pageContext.frontmatter}>
+
+PactFi is including the Degen community as part of its Aeneas Rewards
+<br/>
+
+How much money: 10,000 Algos in rewards
+
+<br/>
+
+When does it start: April 15th
+
+<br/>
+
+How long does it run: 8 weeks
+
+<br/>
+
+[Link to Add Liquidity](https://app.pact.fi/add-liquidity/677288079)
+
+</Post>
```

Comment:
example of a blog post using mdx, notice that I'm using the <Post> component and that component takes in a frontmatter as an argument (props.pageContext.frontmatter). 

Frontmatter is generated from the section at the top that looks like 
```
---
title: 'Blog post title'
date: '01/01/2022'
---